lib/active_model_attributes.rb

```diff
@@ -37,7 +37,10 @@ def define_attribute_reader(name, options)
     def define_attribute_writer(name, cast_type, options)
       wrapper = Module.new do
         define_method "#{name}=" do |val|
-          deserialized_value = ActiveModel::Type.lookup(cast_type, **options.except(*SERVICE_ATTRIBUTES)).cast(val)
+          if cast_type.is_a?(Symbol)
```

Comment:
What is cast type is a string? Will it work with `ActiveModel::Type.lookup`?

Reply:
It's a good question. I took the approach from Rails which does the same thing. I haven't looked if it is just symbols that can be registered as types.

I think it would be better to check if it is a `AciveModel::Type::Value` kind of object instead of symbol to avoid doing the type lookup . I was thinking of trying it for a use case that I had in mind but at the end preferred to maintain compatibility with the upcoming 5.2 release of Rails.

At the end it depends if you want this library to behave the same way as Rails 5.2.

Reply:
Ok, I get the idea here. Looks good to merge 👍